tensorbay/opendataset/CACD/loader.py

```diff
@@ -0,0 +1,104 @@
+#
+# Copyright 2021 Graviti. Licensed under MIT License.
+#
+# pylint: disable=invalid-name
+# pylint: disable=missing-module-docstring
+# pylint: disable=line-too-long
```

Comment:
why we need to disable `line-too-lone`, normally `black` will help format py file and ensure every lines will not exceed 100 characters.